.github/workflows/release-test.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: release-test
+
+on:
+  push:
+    branches:
+      - feature/**
+  workflow_dispatch:
+
+jobs:
+  docker-build:
+    uses: ./.github/workflows/docker-build.yml
+    with:
+      namespace: osis
+      tag: 0.0.27
+    secrets:
+      REGISTRY_LOGIN: ${{ secrets.REGISTRY_LOGIN }}
+      REGISTRY_PASSWORD: ${{ secrets.REGISTRY_PASSWORD }}
+  release:
+    runs-on: ubuntu-latest
+    needs:
+      - docker-build
+    steps:
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          name: Release 0.0.27
+          tag_name: 0.0.27
+          generate_release_notes: true
+          target_commitish: ${{ github.sha }}
```

Comment:
I imagine you're going to delete this file.

Reply:
deleted.